aws_test.go

```diff
@@ -1,4 +1,4 @@
-package ign
+package gz
 
 import (
 	"github.com/stretchr/testify/assert"
```

Comment:
The test below this line is using the `ign` prefix.

Reply:
Good catch, let me take a look at it.